tomviz/ImageStackDialog.h

```diff
@@ -17,8 +17,13 @@
 #ifndef tomvizImageStackDialog_h
 #define tomvizImageStackDialog_h
 
-#include <QDialog>
+#include "DataSource.h"
+#include "ImageStackModel.h"
 
+#include <QDialog>
+#include <QDragEnterEvent>
+#include <QDropEvent>
```

Comment:
Shouldn't need all of these in the header, QDropEvent is just a pointer, I was able to remove QDragEnterEvent, QDropEvent and QMimeData, adding just QDragEnterEvent and QMimeData to the implementation. Usually other classes declare these types, and we can keep them in the implementation when only references/pointers are used in the header.